rules/false_positives/buildah.yara

```diff
@@ -0,0 +1,12 @@
+rule buildah_dev_shm: override {
+  meta:
+    description    = "buildah"
+    dev_shm_hidden = "high"
```

Comment:
I think this should be "low" in this case.

Reply:
Updated in [`ebb4707` (#629)](https://github.com/chainguard-dev/malcontent/pull/629/commits/ebb4707490a4a7ba88f056c7d0298fa716adcbe1).